cmd/pulumi-converter-yaml/main.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// pulumi-converter-yaml is the "language converter" for Pulumi programs written in YAML or JSON. It is
+// responsible for translating JSON/YAML templates into PCL.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/pkg/errors"
```

Comment:
Was this use of pkg/errors intentional?

Reply:
no, probably vscode being overly eager to auto-fill. I'll fix it up.